package-lock.json

```diff
@@ -24686,7 +24695,7 @@
 			"dependencies": {
 				"clone-deep": {
 					"version": "0.2.4",
-					"resolved": "https://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
+					"resolved": "http://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
```

Comment:
Back and forth we go 😵 (I see local changes on `master` when I run `npm install` using the latest version of NPM)

Reply:
😓